source/includes/steps-view-logs.yaml

```diff
@@ -1,20 +1,20 @@
 ---
-title: "Find your Session ID."
+title: "Find your Log ID."
 level: 4
 ref: session-id
 content: |
 
   .. example::
 
-     |mdb-shell| displays the session ID each time you open the shell.
+     |mdb-shell| displays the log ID each time you open the shell.
 
      .. code-block:: javascript
         :copyable: false
         :emphasize-lines: 3
 
         $ mongosh
 
-        Current sessionID: c2961dbd6b73b052671d9df0
+        Current logID: c2961dbd6b73b052671d9df0
```

Comment:
tested from latest build from [evergreen](https://evergreen.mongodb.com/task/mongosh_darwin_codesign_compile_and_upload_artifact_macos_patch_0a629b2b9b60ae76fee2e1a6f47aaea87111876a_5fe35ee057e85a31e9ac1f1d_20_12_23_15_14_41)
```suggestion
        Current Log ID: c2961dbd6b73b052671d9df0
```
![Screen Shot 2020-12-23 at 3 07 22 PM](https://user-images.githubusercontent.com/55147273/103037018-a0981400-4530-11eb-979f-24b7ae0da4f4.png)

Reply:
up to you if you want to make the variables in the rest of the PR consistent with this. Not sure that it matters that much.